SQL/tgstation_schema_prefixed.sql

```diff
@@ -148,6 +148,7 @@ CREATE TABLE `SS13_death` (
   `toxloss` smallint(5) unsigned NOT NULL,
   `cloneloss` smallint(5) unsigned NOT NULL,
   `staminaloss` smallint(5) unsigned NOT NULL,
+  `last_words` varchar(96) DEFAULT NULL
```

Comment:
sizes 255 and below don't change how it's stored or make it use less, so you might as well make it 255